src/client-apps/nuxeo-drive.md

```diff
@@ -242,58 +242,455 @@ Parameter values are taken as is, except for booleans. In that case, you can spe
+* * *
+
+#### `client-version`
+
+Force the client version to run when using the centralized update channel (must be >= `4.2.0`).
+
+- Default value (str): None
+- Version added: 4.2.0
+
+* * *
+
+#### `consider-ssl-errors`
+
+Define if SSL errors should be ignored.
+
+- Default value (bool): True
+- Version added: 2.0
+- Version removed: 4.0.1, use [ssl-no-verify](#ssl-no-verify) set to `True` instead
+
+* * *
+
+#### `database-batch-size`
+
+[Direct Transfer] When adding files into the database, the operation is done by batch instead of one at a time.
+This option controls the batch size.
+
+- Default value (int): 256
+- Version added: 4.4.4
+
+* * *
+
+#### `debug`
+
+Activate the debug window, and debug mode.
+
+- Default value (bool): False
+- Version added: 2.0
+- Version removed: 4.0.0
+
+* * *
+
+#### `delay`
+
+Delay in seconds before each remote check (calling the [NuxeoDrive.GetChangeSummary](https://explorer.nuxeo.com/nuxeo/site/distribution/10.10/viewOperation/NuxeoDrive.GetChangeSummary) operation).
+
+- Default value (int): 30
+- Version added: 2.0
+
+* * *
+
+#### `disabled-file-integrity-check`
+
+Set to `True` to disable downloaded files integrity check.
+It is a needed option when the [managed blob store key strategy](https://doc.nuxeo.com/nxdoc/hotfixes-installation-notes-for-nuxeo-platform-lts-2019/#s3-direct-upload-of-5-gb-files) is set up on the server, because there is no logic digest filled, the application would not be able to validate such files.
+
+- Default value (bool): False
+- Version added: 4.4.5
+
+* * *
+
+#### `disallowed-types-for-dt`
+
+List of document types where Direct Transfer is not allowed.
+
+- Default value (list):
+```python
+[
+    "Domain",
+    "Section",
+]
+```
+- Version added: 4.4.6
+
+* * *
+
+#### `feature`
+
+Application features that can be turned on/off on-demand.
+
+- Default value (map): [...](#features)
+- Version added: 4.4.2
+
+* * *
+
+#### `force-locale`
+
+Force the reset to the language.
+
+- Default value (str): None
+- Version added: 2.0
+
+* * *
+
+#### `handshake-timeout`
+
+Define the handshake timeout in seconds.
+
+- Default value (int): 60
+- Version added: 2.0
+
+* * *
+
+#### `ignored-files`
+
+Lowercase file patterns to ignore while syncing.
+
+- Default value (list):
+```python
+[
+    r"^atmp\d+$",
+]
+```
+- Version added: 2.4.1
+
+* * *
+
+#### `ignored-prefixes`
+
+Lowercase file prefixes to ignore while syncing.
+
+- Default value (list):
+```python
+[
+    ".",
+    "desktop.ini",
+    "icon\r",
+    "thumbs.db",
+    "~$",
+]
+```
+- Version added: 2.4.1
+
+* * *
+
+#### `ignored-suffixes`
+
+Lowercase file suffixes to ignore while syncing.
+
+- Default value (list):
+```python
+[
+    ".bak",
+    ".crdownload",
+    ".dwl",
+    ".dwl2",
+    ".idlk",
+    ".lnk",
+    ".lock",
+    ".nxpart",
+    ".part",
+    ".partial",
+    ".swp",
+    ".tmp",
+    "~",
+]
+```
+- Version added: 2.4.1
+- Version changed: 4.1.0, added `.idlk` (Adobe InDesign lock files)
```

Comment:
For the future, if there is another version changing something, we will move to a sublist. But I think it is better to do that only when needed.